apps/artemis-web/middleware/database.ts

```diff
@@ -15,7 +12,7 @@ export async function setUpDb(db: Db) {
 }
 
 export default async function database(req, res, next) {
-  if (!client.isConnected()) await client.connect();
+  await client.connect();
```

Comment:
here we `await` while in the other instance (remember me strategy) we connect directly. What is the correct approach?

Reply:
The correct is to await :) In practice didnt matter much, but changed the remberme strategy code to be compatible with this